src/spryker/_twig/docker-compose.yml/application.yml.twig

```diff
@@ -0,0 +1,63 @@
+{% set blocks  = '_twig/docker-compose.yml/' %}
+{% set dockersync = false %}
+{% if @('host.os') == 'darwin' and @('docker-sync') == 'yes' %}
+{% set dockersync = true %}
+{% endif %}
+
+  console:
+{% if @('app.build') == 'dynamic' %}
+    build:
+      context: ./
+      dockerfile: .my127ws/docker/image/console/Dockerfile
+    image: {{ @('workspace.name') ~ '-console:dev' }}
+    entrypoint: /entrypoint.dynamic.sh
+    volumes:
+      - {{ (dockersync) ? @('workspace.name') ~ '-sync:/app:nocopy' : '../:/app:delegated' }}
+      - ./.my127ws/application:/home/build/application
+      - ./.my127ws/docker/image/console/root/lib/task:/lib/task
+{% else %}
+    image: {{ @('docker.repository') ~ ':' ~ @('app.version') ~ '-console' }}
+{% endif %}
+    labels:
+      - traefik.enable=false
+    environment: &APP_ENV_VARS
+{% include blocks ~ 'environment.yml.twig' %}
+    networks:
+      - private
+
+  nginx:
+{% if @('app.build') == 'dynamic' %}
+    build: .my127ws/docker/image/nginx
+    volumes:
+      - {{ (dockersync) ? @('workspace.name') ~ '-sync:/app:nocopy' : './:/app:delegated' }}
+{% else %}
+    image: {{ @('docker.repository') ~ ':' ~ @('app.version') ~ '-nginx' }}
+{% endif %}
+    labels:
+      - traefik.backend={{ @('workspace.name') }}
+      - traefik.frontend.rule=Host:{{ @('yves-host') }},{{ @('zed-host') }}
+      - traefik.docker.network=my127ws
+      - traefik.port=80
+    links:
+      - php-fpm:php-fpm
+    networks:
+      private:
+        aliases:
+          - {{ @('hostname') }}
```

Comment:
Should add the yves and zed external hosts here too so that behat can talk to them via the internal network.